src/modules/weather-api.js

```diff
@@ -0,0 +1,25 @@
+const getWeather = () => {
+  async function getWeatherCoor(lat, lon) {
+    const url = `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=ff6c534fa692d1c761f8ca2f57394e60`;
+    const response = await fetch(url, {
+      mode: 'cors',
+      credentials: 'omit',
+    });
+    return response.json();
```

Comment:
Wrap this in a **try** block and add a **catch** block to handle the errors after it.